src/components/DealForm/DealForm.tsx

```diff
@@ -0,0 +1,181 @@
+                    <SingleSelect
+                        id="pipeline"
+                        value={values.pipeline}
+                        options={pipelineOptions}
+                        error={!!(touched.pipeline && errors.pipeline)}
+                        onChange={(value: Option<string>) => setFieldValue("pipeline", value)}
+                    />
+                </Label>
+
+                <Label htmlFor="dealStage" label="Deal stage" required>
+                    <SingleSelect
+                        id="dealStage"
+                        value={values.dealStage}
+                        options={stageOptions}
+                        error={!!(touched.dealStage && errors.dealStage)}
+                        onChange={(value: Option<string>) => setFieldValue("dealStage", value)}
+                    />
+                </Label>
+
+                <AmountField
+                    currency={currency}
+                    error={!!(touched.amount && errors.amount)}
+                    {...getFieldProps("amount")}
+                />
+
+                <DateField
+                    id="closeDate"
+                    label="Close date"
+                    error={!!(touched.closeDate && errors.closeDate)}
+                    onChange={(date: [Date]) => setFieldValue("closeDate", date[0])}
+                />
+
+                <Label htmlFor="dealOwner" label="Deal owner">
+                    <SingleSelect
+                        id="dealOwner"
+                        value={values.dealOwner}
+                        options={ownerOptions}
+                        error={!!(touched.dealOwner && errors.dealOwner)}
+                        onChange={(value: Option<string>) => setFieldValue("dealOwner", value)}
+                    />
+                </Label>
+
+                <Label htmlFor="dealType" label="Deal type">
+                    <SingleSelect
+                        id="dealType"
+                        value={values.dealType}
+                        options={dealTypeOptions}
+                        error={!!(touched.dealType && errors.dealType)}
+                        onChange={(value: Option<string>) => setFieldValue("dealType", value)}
+                    />
+                </Label>
+
+                <Label htmlFor="priority" label="Priority">
+                    <SingleSelect
+                        id="priority"
+                        value={values.priority}
+                        options={priorityOptions}
+                        error={!!(touched.priority && errors.priority)}
+                        onChange={(value: Option<string>) => setFieldValue("priority", value)}
+                    />
+                </Label>
+            </BaseContainer>
+
+            {/*{!isEditMode && <HorizontalDivider />}*/}
+
+            <BaseContainer>
+                {/*
+                {!isEditMode && <Title title="Associate deal with" />}
```

Comment:
You reckon the comments should stay?

Reply:
sorry, I rechecked UI in the Figma and forgot about this. When we create a deal - we need these two fields, but don't need them when updating the deal. I'll fix this now